### DISCUSSION TITLE
[Suggestion]: Dedicated page to share profiles/applications?

### DISCUSSION THREAD

**Discussion by Andre601** [FluxpointDev/MultiRPC#15] [Old]
MultiRPC could benefit from a dedicated page to share and find custom profiles.

To my knowledge are applications used for custom RPC usable by everyone as long as they are public and their ID known, so the site could be as simple as sharing IDs and other info like names for the image keys and maybe a preview of what it may look like.

MultiRPC itself could have a feature where the Add button would allow you to also put an URL to a custom profile on the site and the app would then retrieve the right info from that.

**Comment by xXBuilderBXx** [comment]
Redoing MultiRPC
